command/ibft/snapshot/ibft_snapshot.go

```diff
@@ -19,7 +19,7 @@ func GetCommand() *cobra.Command {
 }
 
 func setFlags(cmd *cobra.Command) {
-	cmd.Flags().IntVar(
+	cmd.Flags().Int64Var(
```

Comment:
Thinking out loud:
should we change this to be `Uint64` type, but treat it like non-mandatory flag. If not provided, we are assuming that latest block snapshot should be retrieved. WDYT, is it feasible to do so?

Reply:
Sounds reasonable and the most straightforward thing to do (I was even thinking about `*uint64`, and somehow setting `nil` if that flag wasn't supplied, but probably it would overcomplicate stuff).

Reply:
Done